Dockerfile

```diff
@@ -1,11 +1,17 @@
 # don't change this line
 FROM korney4eg/nginx-php:latest
 
-# your code goes here
-# Make nginx use app.conf configuration to properly render php files
-# Make sure that index.php file is available in browser
-# expose needed ports
+RUN curl -O https://raw.githubusercontent.com/wp-cli/builds/gh-pages/phar/wp-cli.phar\
```

Comment:
I think this part could be replaced with `ADD` or `COPY`. Could you check it out?

Reply:
Wouldn't I need to store the wp-cli binary locally to do that though? I think I would prefer to download the binary remotely much like how I remotely download the plugin

Reply:
Hey, could you check out documentation for Dockerfile. I belive that ADD command allow you to download from remote url